sdks/python/apache_beam/dataframe/convert.py

```diff
@@ -123,6 +129,8 @@ def to_pcollection(
   pass them all at the same time to this method.
 
   Args:
+    label: (optional, default "ToPCollection(...)"") the label to use for the
+        conversion transform.
```

Comment:
Could you also update the docstring to note this works with non-deferred dataframes, and update the typehint on `dataframes`